met-api/src/met_api/models/user.py

```diff
@@ -24,14 +24,14 @@ class User(db.Model):
     engagement_id = db.relationship('Engagement', backref='user', cascade="all, delete")
 
     @classmethod
-    def find_by_id(cls, id):
+    def find_by_id(cls, _id):
```

Comment:
any reason for making it _id ? prefixed underscores means its kind of private variable